influxql/call_iterator.go

```diff
@@ -781,39 +781,47 @@ func IntegerSpreadReduceSlice(a []IntegerPoint) []IntegerPoint {
 	return []IntegerPoint{{Time: ZeroTime, Value: max - min}}
 }
 
-func newTopIterator(input Iterator, opt IteratorOptions, n int) (Iterator, error) {
+func newTopIterator(input Iterator, opt IteratorOptions, n int, writeMode bool) (Iterator, error) {
```

Comment:
Should `writeMode` be named `keepTags`?  Seems like that is what it indicates as opposed to having multiple values.